client/my-sites/patterns/components/pattern-library/index.tsx

```diff
@@ -34,15 +31,17 @@ import {
 import { useSelector } from 'calypso/state';
 import { isUserLoggedIn } from 'calypso/state/current-user/selectors';
 import getUserSetting from 'calypso/state/selectors/get-user-setting';
+import { usePatternsContext } from '../../context';
 import { getTracksPatternType } from '../../lib/get-tracks-pattern-type';
+import { PatternsSearchField } from '../search-field';
```

Comment:
```suggestion
import { usePatternsContext } from 'calypso/my-sites/patterns/context';
import { getTracksPatternType } from 'calypso/my-sites/patterns/lib/get-tracks-pattern-type';
import { PatternsSearchField } from 'calypso/my-sites/patterns/components/search-field';
```
Let's stick with absolute import paths